playwright/e2e/rosa-hosted/rosa-cluster-hosted-machine-pool-cr.spec.ts

```
@@ -0,0 +1,144 @@
+import { test, expect } from '../../fixtures/pages';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)